sw/device/lib/dif/dif_keymgr_dpe.h

```diff
@@ -85,6 +85,10 @@ typedef struct dif_keymgr_dpe_advance_params {
  * peripheral port to which the generated key is loaded.
  */
 typedef enum dif_keymgr_dpe_key_dest {
```

Comment:
The commit message here does not match the content (which only adds a `dif_keymgr_dpe_key_dest`)